src/main/java/com/example/javaOjt/services/AuthorService.java

```diff
@@ -57,4 +59,16 @@ public GetAuthorResponse getAuthorById(Integer id, boolean withBooks) {
     return response;
   }
 
+  /**
+   * Fetches the complete list of authors.
+   */
+  public List<GetAuthorSummaryResponse> getAuthorList() { // Should return an empty list on empty DB
+    List<GetAuthorSummaryResponse> response = new ArrayList<>();
+    Iterable<Author> authors = authorRepository.findAll();
+    for (Author author : authors) {
+      response.add(new GetAuthorSummaryResponse(author));
+    }
```

Comment:
`forEach()` を使うともっと簡潔！

```suggestion
    authorRepository.findAll().forEach(author -> 
      response.add(new GetAuthorSummaryResponse(author))
    );
```